.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  useTabs: false,
+  plugins: [require("prettier-plugin-jinja-template")],
+  overrides: [
+    {
+      files: ["**/{templates,jinja2}/**/*.html"],
```

Comment:
shouldn't it be just? 
```suggestion
      files: ["*.html"],
```
that's what jinja-template plugin recommends and I'm not sure if we're going to have any pure HTML files

Reply:
agree